packages/cli/src/bin/zos-cli.ts

```diff
@@ -3,11 +3,15 @@
 import { Logger } from 'zos-lib';
 import { lockSync } from 'lockfile';
 import program from './program';
+import lookForRootDirectory from './helpers';
+
+const rootDirectory = lookForRootDirectory(process.cwd()) || process.cwd();
+if (rootDirectory !== null && rootDirectory !== process.cwd()) process.chdir(rootDirectory);
 
 // Acquire file lock to ensure no other instance is running
 const LOCKFILE: string = '.zos.lock';
 try {
-  lockSync(LOCKFILE, { retries: 0 });
+  lockSync(`${rootDirectory}/${LOCKFILE}`, { retries: 0 });
```

Comment:
```suggestion
  lockSync(LOCKFILE, { retries: 0 });
```
Should not be needed, as we are doing a chdir right before